windows/src/global/delphi/general/kmpinffile.pas

```diff
@@ -162,6 +162,11 @@ procedure TKMPInfFile.Import(AIni: TIniFile);
     //if LowerCase(AIni.ReadString('Install', 'StartMenuReadme', '')) = 'true' then -- already in [StartMenu]
     Options.ReadmeFile := Files.FromFileName(AIni.ReadString('Install', 'ReadmeFile', ''));
     Options.ExecuteProgram := AIni.ReadString('Install', 'ExecuteProgram', '');
+
+//    TODO
+//    Also remember to update version number if source files change
+//    Also do not permit different versions of same keyboard in compiled form!
```

Comment:
Does this have an issue, if we're leaving it to a future PR?  Or was this accidentally left out?

Reply:
This will be in a future PR. I should pull it out of the source code and put it into the trello card.